backend/src/main/java/org/cryptomator/hub/api/UsersResource.java

```diff
@@ -158,7 +160,7 @@ public UserDto getMe(@QueryParam("withDevices") boolean withDevices) {
 		User user = userRepo.findById(jwt.getSubject());
 		Function<Device, DeviceResource.DeviceDto> mapDevices = d -> new DeviceResource.DeviceDto(d.getId(), d.getName(), d.getType(), d.getPublickey(), d.getUserPrivateKeys(), d.getOwner().getId(), d.getCreationTime().truncatedTo(ChronoUnit.MILLIS));
 		var devices = withDevices ? user.devices.stream().map(mapDevices).collect(Collectors.toSet()) : Set.<DeviceResource.DeviceDto>of();
-		return new UserDto(user.getId(), user.getName(), user.getPictureUrl(), user.getEmail(), user.getLanguage(), devices, user.getEcdhPublicKey(), user.getEcdsaPublicKey(), user.getPrivateKeys(), user.getSetupCode());
+		return new UserDto(user.getId(), user.getName(), user.getPictureUrl(), user.getEmail(), user.getLanguage(), devices, user.getEcdhPublicKey(), user.getEcdhPublicKey(), user.getEcdsaPublicKey(), user.getPrivateKeys(), user.getPrivateKeys(), user.getSetupCode());
```

Comment:
`user.getEcdhPublicKey(), user.getEcdhPublicKey()` and `user.getPrivateKeys(), user.getPrivateKeys()` looks terrible.

Instead, now that we have marked one constructor for JSON deserialization, we should be fine adding a second constructor using just the "new" standard.

Reply:
Fixed in [349f520](https://github.com/cryptomator/hub/pull/315/commits/349f52083f95e6b78e0e9842c31ca84697ba89e4)